specs/current/ptc/scripts/ptc_debug_calling.py

```
@@ -234,14 +283,17 @@ def main():
         (call_id, tool_name, status, inp, created_at, ptc_call_id)
         for call_id, tool_name, tc_type, status, inp, out, error, created_at, ptc_call_id in raw
         if ((tc_type == "ptc") if tc_type else call_id.startswith("ptc:"))
-        and ptc_call_id and ptc_call_id not in all_agent_ids
+        and ptc_call_id
+        and ptc_call_id not in all_agent_ids
     ]
     if orphan_ptc:
         print(f"  [Orphan PTC calls — parent not found] ({len(orphan_ptc)})")
         for call_id, tool_name, status, inp, created_at, ptc_call_id in orphan_ptc:
```

Ruff (0.15.2)

[warning] Loop control variable `call_id` not used within loop body

Rename unused `call_id` to `_call_id`

(B007)